### PR TITLE
Append2

### PR DESCRIPTION
**Pull request by goelvatsal**

### CODE REVIEW

**Comment by goelvatsal** [comment]
11-housing-prices is in WIP

**Comment by goelvatsal** [comment]
I accidentally commited

File: 16-slices/exercises/10-append-sort-nums/main.go [hunk 1]
```diff
@@ -43,4 +50,25 @@ package main
...
+		if i == 0 {
+			continue
+		}
+		s = append(s, v)
+	}
+
+	var i []int
+	for _, v := range s {
+		f, _ := strconv.Atoi(v)
+		i = append(i, f)
```
Comment (goel4607) [inline 16-slices/exercises/10-append-sort-nums/main.go:69]: handle the err case here

File: 16-slices/exercises/10-append-sort-nums/main.go [hunk 2]
```diff
@@ -43,4 +50,25 @@ package main
...
+
+	var s []string
+	for i, v := range os.Args {
+		if i == 0 {
+			continue
+		}
+		s = append(s, v)
+	}
+
+	var i []int
```
Comment (goel4607) [inline 16-slices/exercises/10-append-sort-nums/main.go:66]: never use the variable name as i until and unless it is the index in a loop; always use good small idiomatic names like nums (small for numbers)

**Review by goel4607 — APPROVED** [review]

**Pull request merged**